schemas/mod-entities-links/linkingRuleDto.json

```diff
@@ -74,6 +61,10 @@
           }
         }
       }
+    },
+    "bibRecordTag": {
+      "type": "string",
+      "description": "Bibliographic record tag"
```

Comment:
Is it not the same as `bibField` ?

Reply:
Actually, it was the same field, so removed.